src/components/media-player/media-player-track.js

```diff
@@ -209,39 +217,41 @@ function MediaPlayerTrack({
   return (
     <div className={classes}>
       <div className="MediaPlayerTrack-aspectRatioContainer" style={{ paddingTop }}>
-        <ReactPlayer
-          ref={playerRef}
-          className={playerScreenClasses}
-          url={sourceUrl}
-          width="100%"
-          height="100%"
-          controls={false}
-          volume={volume}
-          muted={volume === 0}
-          playbackRate={playbackRate}
-          progressInterval={MEDIA_PROGRESS_INTERVAL_IN_MILLISECONDS}
-          light={loadImmediately ? false : trackPlayState.current === MEDIA_PLAY_STATE.initializing && (posterImageUrl || true)}
-          playing={shouldPlay}
-          onReady={handleReady}
-          onBuffer={handleBuffer}
-          onBufferEnd={handleBufferEnd}
-          onStart={handlePlay}
-          onPlay={handlePlay}
-          onPause={handlePause}
-          onEnded={handleEnded}
-          onDuration={handleDuration}
-          onProgress={handleProgress}
-          onClickPreview={handleClickPreview}
-          />
+        {!!isMounted.current && (
```

Comment:
We need to do this because the react player renders into having a difference between server side and client side 😬

Reply:
:grimacing: Let's just hope there are not even worse breaking changes